Gopkg.lock

```diff
@@ -333,6 +333,12 @@
   packages = ["."]
   revision = "e181e095bae94582363434144c61a9653aff6e50"
 
+[[projects]]
```

Comment:
Should there be a corresponding change to add "fasthash" to Gopkg.toml?

Reply:
it wouldn't actually have much effect at the moment, as fasthash has never published a release; literally the only thing we could put is `branch - "master"`. Fine to go without in this case, IMO.